aicsimageio/writers/ome_zarr_writer.py

```diff
@@ -249,26 +266,39 @@ def write_image(
             * image_data.shape[ydimindex]
             * image_data.itemsize
         )
+
         target_chunk_size = 16 * (1024 * 1024)  # 16 MB
         # this is making an assumption of chunking whole XY planes.
         # TODO allow callers to configure chunk dims?
         nplanes_per_chunk = int(math.ceil(target_chunk_size / plane_size))
         nplanes_per_chunk = (
             min(nplanes_per_chunk, image_data.shape[zdimindex]) if zdimindex > -1 else 1
         )
-        chunk_dim_map = {
-            "T": 1,
-            "C": 1,
-            "Z": nplanes_per_chunk,
-            "Y": image_data.shape[ydimindex],
-            "X": image_data.shape[xdimindex],
-        }
-        chunk_dims = [
-            dict(
-                chunks=tuple(chunk_dim_map[d] for d in dimension_order),
-                compressor=default_compressor,
-            )
-        ]
+
```

Comment:
lines 271-277 should be moved inside the if statement below?

Reply:
also the comment on line 272 can now go away because callers CAN configure chunk dims

Reply:
resolved in [91b2d97](https://github.com/AllenCellModeling/aicsimageio/pull/519/commits/91b2d97632eff12c31a88d4fbef4962be04b9171)